source/common/grpc/async_client_manager_impl.h

```diff
@@ -66,6 +66,7 @@ class AsyncClientManagerImpl : public AsyncClientManager {
                   const RawAsyncClientSharedPtr& client);
 
     RawAsyncClientSharedPtr getCache(const GrpcServiceConfigWithHashKey& config_with_hash_key);
+    int32_t timer_enabled_with_0_duration_count_ = 0;
```

Comment:
you shouldn't need this just for test. if you want to test you should use a mock timer.